src/main/java/org/opentripplanner/middleware/utils/HttpResponseHandler.java

```diff
@@ -0,0 +1,35 @@
+package org.opentripplanner.middleware.utils;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.util.EntityUtils;
```

Comment:
Is this a transitive dependency? Maybe the Apache HttpCore should be included as a dependency in the pom.xml.

Reply:
It is not. A number of dependencies include it as a sub-dependency. It is now a top level dependency.